core/src/data/value/binary_op/i64.rs

```diff
@@ -295,6 +352,8 @@ mod tests {
         let type_max: i64 = i64::MAX;
         let type_min: i64 = i64::MIN;
         let type_maxi64: i64 = type_max;
+        let type_maxi128: i128 = type_max.into();
+        let type_mini128: i128 = type_min.into();
```

Comment:
Since this is a typecast from `i64::MAX` , it looks different from the intended `max_i128` value.

Reply:
`Into` is a trait for transforms. Now, since we have converted the i64 variable to i128, the value I inferred from the variable name is the same as `i128::MAX` or `MIN`. How about like this?

Reply:
Do you mean something like maxI64intoi128 as a variable name?

Reply:
How about.. not to use variables? explicit use of `i64::MAX` or `(i64::MAX as i128)` would be better in these cases.

```rust
let type_max: i64 = i64::MAX;
// i64:MAX

let type_min: i64 = i64::MIN;
// i64::MIN

let type_maxi64: i64 = type_max;
// i64::MAX

let type_maxi128: i128 = type_max.into();
// (i64::MAX as i128) or i64::MAX.into()

let type_mini128: i128 = type_min.into();
// (i64::MIN as i128) or i64::MIN.into()
```